SConstruct

```diff
@@ -263,7 +263,7 @@ config_options = [
            will be installed to the system default 'site-packages' directory.
            To install to the current user's 'site-packages' directory, use
            'python_prefix=USER'.""",
-        {"Windows": "", "default": "$prefix"},
+        {"default": ""},
```

Comment:
I think this change will work out because `pip` does the right thing at install time, whereas pure Setuptools needed a little more hand-holding. But I didn't get a chance to test on Windows, and I'd appreciate at least some ad-hoc testing on your guys' end.

Reply:
It does.